src/credentials/attestation_verifier/TestDACRevocationDelegateImpl.cpp

```diff
@@ -53,6 +82,59 @@ void TestDACRevocationDelegateImpl::ClearDeviceAttestationRevocationSetPath()
     mDeviceAttestationRevocationSetPath = mDeviceAttestationRevocationSetPath.substr(0, 0);
 }
 
+CHIP_ERROR TestDACRevocationDelegateImpl::GetSubjectAndKeyIdFromPEMCert(const std::string & certPEM, std::string & outSubject,
+                                                                        std::string & outKeyId)
+{
+    // buffers and spans for storing crl signer delegator OR crl signer cert info
+    char subjectBuf[kMaxIssuerBase64Len]             = { 0 };
+    char skidBuf[2 * kAuthorityKeyIdentifierLength]  = { 0 };
+    uint8_t certDerBuf[kMax_x509_Certificate_Length] = { 0 };
+
+    MutableCharSpan subject(subjectBuf);
+    MutableCharSpan keyId(skidBuf);
+    MutableByteSpan certDER(certDerBuf);
+
+    ReturnLogErrorOnFailure(X509_PemToDer(certPEM, certDER));
+    ReturnErrorOnFailure(GetSubjectNameBase64Str(certDER, subject));
+    ReturnErrorOnFailure(GetSKIDHexStr(certDER, keyId));
+
+    outSubject = std::string(subject.data(), subject.size());
+    outKeyId   = std::string(keyId.data(), keyId.size());
+
+    return CHIP_NO_ERROR;
+}
+
+// cross validate DAC/PAI cert details with crl signer OR crl signer delegator
+bool TestDACRevocationDelegateImpl::CrossValidateCert(const Json::Value & revokedSet, const CharSpan & akidHexStr,
+                                                      const CharSpan & issuerNameBase64Str)
+{
+    std::string certPEM;
+    const char * certType __attribute__((unused));
```

Comment:
Why is `__attribute__((unused))` there?  If compilers complain when detail logging is enabled, then please use standard `[[maybe_unused]]` and not compiler extensions.

Reply:
Replaced it with `[[maybe_unused]]`